bin/crew

```diff
@@ -1811,12 +1811,69 @@ def update_sha256(package, key, value)
+    puts "\nCopied #{CREW_LOCAL_REPO_ROOT}/packages/#{pkg_name}.rb to #{CREW_PACKAGES_PATH}.\n".lightgreen
+  else
+    # This pulls the operation from the calling function
+    operation = caller(1, 2)[1].to_s.split[3].split('_')[0]
+    puts prompt_msg.yellow
+    print "\nWould you like to copy #{pkg_name}.rb to crew and start the #{operation}? [Y/n] ".yellow
+    response = $stdin.gets.chomp.downcase
+    case response
+    when 'n', 'no'
+      puts "#{operation.capitalize} skipped."
+      next_pkg = true
+    when '', 'y', 'yes'
+      FileUtils.cp "#{CREW_LOCAL_REPO_ROOT}/packages/#{pkg_name}.rb", "#{CREW_PACKAGES_PATH}/"
+      puts "\nCopied #{CREW_LOCAL_REPO_ROOT}/packages/#{pkg_name}.rb to #{CREW_PACKAGES_PATH}.\n".lightgreen
+    else
+      puts "I don't understand `#{response}`. :(".lightred
+      puts "#{operation.capitalize} skipped."
+      next_pkg = true
+    end
+  end
+  return next_pkg
+end
+
+def check_package(pkg_name)
+  next_pkg = nil
+  return copy_package(pkg_name) if @opt_force
+
+  # Prompt to copy the local repo package to crew if the package is not found.
+  if !File.file?("#{CREW_PACKAGES_PATH}/#{pkg_name}.rb") && File.file?("#{CREW_LOCAL_REPO_ROOT}/packages/#{pkg_name}.rb")
+    prompt_msg = "\n#{CREW_LOCAL_REPO_ROOT}/packages/#{pkg_name}.rb has been updated but the crew package is not found."
+    next_pkg = copy_package(pkg_name, prompt_msg)
+  end
+
+  # Compare local repo package to the crew repo package and prompt to copy if necessary to prepare for the operation.
+  crew_package_updated = ''
+  Dir.chdir CREW_PACKAGES_PATH do
+    crew_package_updated = `git diff #{CREW_PACKAGES_PATH}/#{pkg_name}.rb`.chomp
+  end
+  local_package_updated = ''
+  Dir.chdir CREW_LOCAL_REPO_ROOT do
+    local_package_updated = `git diff #{CREW_LOCAL_REPO_ROOT}/packages/#{pkg_name}.rb`.chomp
+  end
+  if local_package_updated != '' && crew_package_updated == ''
+    prompt_msg = "\n#{CREW_LOCAL_REPO_ROOT}/packages/#{@pkgName}.rb has been updated but the crew package is unchanged."
+    next_pkg = copy_package(pkg_name, prompt_msg)
+  end
+  if local_package_updated != '' && crew_package_updated != '' && local_package_updated != crew_package_updated
+    prompt_msg = "\n#{CREW_LOCAL_REPO_ROOT}/packages/#{@pkgName}.rb has been updated and does not match the crew updated package."
+    next_pkg = copy_package(pkg_name, prompt_msg)
+  end
+  return next_pkg
+end
+
 def build_command(args)
   abort 'Unable to locate local repo root directory. Change to a local chromebrew git repo directory and try again.'.lightred unless Dir.exist? CREW_LOCAL_REPO_ROOT
   abort 'Change to a local chromebrew git repo directory and try again.'.lightred if CREW_PACKAGES_PATH.include?(CREW_LOCAL_REPO_ROOT)
   unless Dir.exist? CREW_LOCAL_BUILD_DIR
     if @opt_force
-      puts "Attempting to create local build directory at #{CREW_LOCAL_BUILD_DIR} .".orange
+      puts "Attempting to create local build directory at #{CREW_LOCAL_BUILD_DIR}.".orange
```

Comment:
There is a space here so that the path doesn't end in a `.` - Is that bad?

Reply:
No but maybe `...` makes more sense?